nwbwidgets/ophys.py

```diff
@@ -160,20 +161,27 @@ def __init__(self, plane_seg: PlaneSegmentation, color_wheel=color_wheel, **kwar
         self.categorical_columns = infer_categorical_columns(plane_seg)
         self.plane_seg = plane_seg
         self.color_wheel = color_wheel
-
+        self.progress_bar = ProgressBar()
+        self.button = widgets.Button(description='display Image Masks')
+        self.children = [self.progress_bar.container, self.button]
+        self.button.on_click(self.on_button_click)
+        self.kwargs = kwargs
+
+    def on_button_click(self, b):
```

Comment:
created a button serves two purposes:
1) without this, the progress bar does not show up since jupyter notebook does not show the bar until the figure is populated with all the image_masks (which is what the progress bar tracks).
2) to prevent the notebook from freezing if the user by mistake clicks on the planesegmentation dropdown.